assignment-09-challenge.Rmd

```diff
@@ -1,6 +1,8 @@
 ---
 title: "Challenge assignment (14 marks)"
-output: html_notebook
+output:
+  pdf_document: default
+  html_notebook: default
```

Comment:
So the Travis-breaking error message looks like this:

```
Error in (function (toc = FALSE, toc_depth = 3, toc_float = FALSE, number_sections = FALSE,  : 
  The lib_dir option is not valid for the html_notebook format.
Error: callr subprocess failed: The lib_dir option is not valid for the html_notebook format.
```

I feel like we've dealt with this before, but I forget what the fix was. Did removing `html_notebook` altogether do it?

Reply:
Yeah I think the yml needs the `html_document: Default` to be removed